be/example/src/config.ts

```diff
@@ -4,9 +4,19 @@ export const FE_ORIGIN = 'http://dev.seekunifiedbeta.com:8080';
 
 export const USER_AGENT = 'wingman/local';
 
-// Take care when handling your token; it’s easy for environment variables to be
-// accessed by other processes, accidentally logged, or persisted to shell
-// history. We recommend wiring this up to a secret store like AWS Secrets
-// Manager.
+/**
+ * Take care when handling your token; it’s easy for environment variables to be
+ * accessed by other processes, accidentally logged, or persisted to shell
+ * history. We recommend wiring this up to a secret store like AWS Secrets
+ * Manager.
+ */
 export const getPartnerToken = () =>
   Promise.resolve(process.env.DANGEROUS_PLAYGROUND_TOKEN ?? '');
+
+/**
+ * This ignores signature verification for all incoming webhooks! Don’t do this
```

Comment:
@zsims has really gotten to you. Should we strengthen the language on the Developer Site to encourage signature verification?

Reply:
Signed webhooks vs unsigned webhooks is the new webhooks vs polling